var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -92,18 +79,18 @@ class Coreneuron(CMakePackage):
     # sympy, codegen and ispc options are only usable with nmodl
     conflicts('+sympyopt', when='~sympy')
     conflicts('+sympy', when='~nmodl')
-    conflicts('+sympy', when='coreneuron@0.17')  # issue with include directories
     conflicts('+codegenopt', when='~nmodl')
     conflicts('+ispc', when='~nmodl')
 
-    # Caliper instrumentation is only supported after 1.0.1
-    conflicts('+caliper', when='@:1.0.1')
+    # Caliper instrumentation is only supported after 1.0.0.20210519
+    # Note: The 20210518 date is needed to specify a version before 20210519!
+    conflicts('+caliper', when='@:1.0.0.20210518')
 
     # An old comment said "PGI compiler not able to compile nrnreport.cpp when
     # enabled OpenMP, OpenACC and Reporting. Disable ReportingLib for GPU", but
     # with the contemporary develop version it seems to work. Encode this
     # knowledge as a conflict between +report and +gpu for older versions.
-    conflicts('+report', when='coreneuron@:1.0b+gpu+openmp')
+    conflicts('+report', when='coreneuron@:0.22+gpu+openmp')
```

Comment:
@pramodk, I assume the official release v1.0 and later versions does not reproduce this issue, right?

Reply:
I think that I wrote the above comment and that at the time (post 1.0b, pre 1.0 release) then develop worked. So I think what you've done makes sense.